vtex-apps/shipping-option-components
# feat: add popover to delivery button

#### What problem is this solving?
This Pr adds a popover to location button
card: [IS-5655](https://vtex-dev.atlassian.net/browse/IS-5655)

#### How to test it?

Open the page and a popover should open

[Workspace](https://arthurdp--biggy.myvtex.com/?__bindingAddress=biggy.myvtex.com/)

#### Screenshots or example usage:

https://github.com/user-attachments/assets/9b379ee3-ad1a-40f6-b4f6-d284d0b31c95



#### Describe alternatives you've considered, if any.



#### Related to / Depends on



#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExYXQ3dnN4MGg3Y2NsZHdzdWhzcHM2cmV4aGRiZWUxdnlxMXppNXdwdCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/xSM46ernAUN3y/giphy.gif)


[IS-5655]: https://vtex-dev.atlassian.net/browse/IS-5655?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
